### ISSUE TITLE
fetchMore has no support for fragments | missing parameter

### ISSUE THREAD

**Issue by seeden** [apollostack/apollo-client#832]
I am trying to do fetchMore with fragments. But without any success
I tried next syntax but it is not supported

```js
return fetchMore({
   query,
   fragments,
   variables: {
     cursor,
   },
   updateQuery,
});
```

Am I missing something?

**Comment by seeden** [comment]
I am moving this to react-apollo

**Comment by stubailo** [comment]
That's surprising! I think it was just overlooked, and should be really easy to fix.

**Comment by helfer** [comment]
Should be fixed with #842
